cw_core/lib/digest_auth.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:http/http.dart' as http;
+import 'package:http/io_client.dart' as ioc;
+
+class DigestAuth{
+  Future<http.Response>request({
+        String uri, 
+        String login = "", 
+        String password = "",
+      }) async {
+    final path = '/json_rpc';
+    final rpcUri = Uri.http(uri, path);
+    final realm = 'monero-rpc';
+    final postMap = {
```

Comment:
Avoid to use types suffix for naming of variables if that possible. For that case better to rename the variable to `body` as from your code is clean understable that it will be used for `post` request.

Reply:
fixed